### ISSUE TITLE
Old OpenSSL library version with vulnerability (Google says it can remove the app from Play Store)

### ISSUE THREAD

**Issue by xtazz** [yixia/VitamioBundle#248]
Hello!

I’m developing an app for Android and using Vitamio video player in it. I’ve uploaded my app for internal testing in Google Play and it says that my app can be removed from Google Play because I use the old version of OpenSSL (with recently found shellshock vulnerability). The only SDK I’m using in the app is Vitamio SDK, so that means that the old version of OpenSSL is inside Vitamio library and I can’t remove it because it’s compiled and I can’t remove OpenSSL from there. Can you help me with this issue? Maybe you can rebuild your SDK without OpenSSL or with latest version. Thank you very much in assistance!

**Comment by Akiracr** [comment]
I think i have the same problem, Didi you solved the problem?

**Comment by xtazz** [comment]
@Akiracr Yes, I did. You can download updated binary version from Vitamio here: https://github.com/yixia/VitamioBundle/tree/master/vitamio/res/raw
But they only updated it for ARM, not x86
